### ISSUE TITLE
Make the times more user friendly

### ISSUE THREAD

**Issue by dlespiau** [dlespiau/patchwork#3]
Instead of displaying 2015-03-16, we should probably say "March 16th", "Mar 16" or similar. Maybe just "Tuesday 4:32pm" for the current week, "4:32pm" for the same day, etc...

**Comment by jnikula** [comment]
Personally I'd put this one far down on the priority list. "User friendly" becomes terribly subjective when you add locale/language specific things into the mix. (For one, 12 hour clock with am/pm is dreadful from my POV.)

Instead, please consider adding "N weeks/days/hours ago" next to the ISO format date/time.

**Comment by dlespiau** [comment]
Absolutely! that's indeed what I had in mind these days, relative dates. As you say though, low priority.
